listen_to_mqtt.sh

```diff
@@ -66,126 +71,157 @@ listen_to_mqtt() {
           ;;
 
         *)
-          log_error "Invalid configuration request; topic:$topic vin:$vin msg:$msg"
+          log_error "Invalid configuration command request; vin:$vin topic:$topic msg:$msg"
           ;;
         esac
         ;;
 
       command)
         case $msg in
-        wake)
-          log_notice "Waking Up"
-          send_command $vin "-domain vcsec $msg"
+        autosecure-modelx)
+          send_command $vin $msg "Close falcon-wing doors and lock vehicle"
```

Comment:
Ok, So you put logging only in send_command 👍
Did not test the Cybertruck/model x specific commands... Obviously

Reply:
Ultimately it would be much better (I think) to have a tesla-control-commands.json, which is more or less the output of `tesla-control --help` which contains all the cmd_name... This list of cases is a bit worrying to me and not very much automatic :)

Reply:
Do you read in my mind while I sleep?  I thought too take that output too from tesla-command, it helps but can't be blindly done. Also ... ~95-98% of the job is already done... would have been nice to do this right from the beginning.. but would still need a human to finalize the change.

Another idea... if you look at [commands.go](https://github.com/teslamotors/vehicle-command/blob/main/cmd/tesla-control/commands.go) , it's easy to grep all commands. We could keep in our repo a copy from the "grep commands" and in a githook when we pick a change; automatically create a branch, commit the diff and create a PR.